modules/enabling-scanner-v4-helm.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-helm_{context}"]
+= Enabling RHACS Scanner V4 when installing with Helm
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable it during installation.
+
+If you are installing {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Run the command to install {product-title-short} with customizations, following the instructions in "Install Central using Helm charts with customizations":
++
+[source,terminal]
+----
+helm install -n stackrox --create-namespace \
```

Comment:
Similar comments/suggestions to add "following" to the step language and to add a $ prompt to the command, please apply this same feedback anywhere else it applies in the PR:
```suggestion
. Run the following command to install {product-title-short} with customizations, following the instructions in "Install Central using Helm charts with customizations":
+
[source,terminal]
----
$ helm install -n stackrox --create-namespace \
```